db/seeds.rb

```diff
@@ -1,9 +1,24 @@
+require 'csv'
 # Create admin user
-begin 
-  User.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password', admin: true) if Rails.env.development?
+begin
+  if Rails.env.development?
+    admin = User.new
+    admin.email = "admin@example.com"
+    admin.password = "password"
+    admin.name = "Mayflower Admin"
+    admin.admin = true
+    admin.save
+
+    staff = User.new
+    staff.email = "staff@example.com"
+    staff.password = "password"
+    staff.name = "Mayflower Staff"
+    staff.admin = false
+    staff.save
+  end
```

Comment:
Also, you appear to have forgotten how you defined the `admin` setter function; neither of these users will be admins because, according to the User model, you have to set it as:
`user.admin = 'Admin'`
That's pretty confusing, and I would suggest you change the model rather than the lines here.

Reply:
^^^ about this last comment, I realize you made it a string instead of a Boolean because of the new user view Staff/Admin selector. If you really want to keep it as a string, I'd be ok with it only because you have some code that slightly benefits from it; but the view/controller could be adjusted to work with bools as well.

Reply:
I realize that I seeded my personal database before making some changes regarding the setter and password confirm. Sorry about that!